src/main/scala/org/constellation/primitives/Transaction.scala

```diff
@@ -71,18 +96,40 @@ case class Transaction(edge: Edge[TransactionEdgeData]) {
 
 }
 
-sealed trait TransactionValidation {
+sealed trait TransactionValidationError {
   def errorMessage: String
 }
 
-case class HashDuplicateFoundInSnapshot() extends TransactionValidation {
+case class HashDuplicateFoundInSnapshot() extends TransactionValidationError {
   def errorMessage: String = "Transaction hash already exists in old data"
 }
 
-case class HashDuplicateFoundInRecent() extends TransactionValidation {
+case class HashDuplicateFoundInRecent() extends TransactionValidationError {
   def errorMessage: String = "Transaction hash already exists in recent data"
 }
 
-case class InvalidSourceSignature() extends TransactionValidation {
+case class InvalidSourceSignature() extends TransactionValidationError {
   def errorMessage: String = s"Transaction has invalid source signature"
 }
+
+case class TransactionSerialized(
+  hash: String,
+  sender: String,
+  receiver: String,
+  amount: Long,
+  signers: Set[String],
+  time: Long
```

Comment:
can you give a more descriptive name for time? is this time it was serialized? created?